tests/Checkout/CheckoutDeliveryTest.php

```diff
@@ -197,39 +196,35 @@ public function a_country_must_exist()
             ->assertSessionHasErrors('country');
     }
 
-    /** @test */
-    public function a_country_cant_be_to_long()
-    {
-        $data = $this->createUserData('country', str_repeat('a', 51));
-        $this->post(route('butik.checkout.delivery'), (array)$data)
-            ->assertSessionHasErrors('country');
-    }
-
     /** @test */
     public function if_a_new_country_has_been_selected_the_view_will_be_reloaded()
     {
-        create(Country::class, ['slug' => 'spain']);
+        $country_code = 'ES';
 
         $deliveryRoute = route('butik.checkout.delivery');
 
         // view the form
         $this->get($deliveryRoute);
 
         // submit the form
-        $this->post($deliveryRoute, (array)$this->createUserData('country', 'spain'))
+        $this->post($deliveryRoute, (array)$this->createUserData('country', $country_code))
             ->assertRedirect($deliveryRoute);
     }
 
     /** @test */
-    public function if_a_new_country_has_been_selected_it_will_be_saved_in_the_card()
+    public function if_a_new_country_has_been_selected_it_will_be_saved_in_the_cart()
     {
+        // TODO: This contradicts CheckoutController@saveCustomerData
```

Comment:
I did miss that the names will already be translated. Great!

The redirect is meant as feature. If the country changes, it's possible that some products are not available anymore and that prices do change as well as shipping costs. To make the new prices, availability etc. 100 % clear, the redirect exist. 

To make it even more clear, it could be a thought to redirect with a flash message saying »the order has been updated« or similar.

Reply:
In this case, this test has to assert that a redirection happens.